vmdb/app/controllers/container_controller.rb

```diff
@@ -0,0 +1,360 @@
+      action = "container_edit"
+    else
+      action = nil
+    end
+    return partial,action,header
+  end
+
+  # Replace the right cell of the explorer
+  def replace_right_cell(action = nil, replace_trees = [])
+    @explorer = true
+    partial, action_url, @right_cell_text = set_right_cell_vars(action) if action # Set partial name, action and cell header
+    get_node_info(x_node) if !@in_a_form && !params[:display]
+    replace_trees = @replace_trees if @replace_trees  # get_node_info might set this
+    type, _ = x_node.split("_").last.split("-")
+    trees = {}
+    if replace_trees
+      trees[:containers] = build_containers_tree if replace_trees.include?(:containers)
+    end
+    record_showing = type && ["Container"].include?(TreeBuilder.get_model_for_prefix(type))
+    if !@in_a_form && !@sb[:action]
+      v_buttons, v_xml = build_toolbar_buttons_and_xml("x_gtl_view_tb") unless record_showing
+      c_buttons, c_xml = build_toolbar_buttons_and_xml(center_toolbar_filename)
+    end
+    h_buttons, h_xml = build_toolbar_buttons_and_xml("x_history_tb") unless @in_a_form
+
+    render :update do |page|
+      # Build hash of trees to replace and optional new node to be selected
+      trees.each do |tree_name, tree|
+        page.replace(
+          "#{tree_name}_tree_div",
+          :partial => 'shared/tree',
+          :locals  => {
+            :tree => tree,
+            :name => tree.name
+          }
+        )
+      end
+      page << "dhxLayoutB.cells('b').setText('#{escape_javascript(h(@right_cell_text))}');"
+
+      # Replace right cell divs
+      if ["container_edit"].include?(action)
+        page.replace_html("main_div", :partial=>partial)
+      elsif params[:display]
+        partial_locals = Hash.new
+        partial_locals[:controller] = "container"
+        partial = "layouts/x_gtl"
+        if partial == "layouts/x_gtl"
+          partial_locals[:action_url] = @lastaction
+          page << "miq_parent_id = '#{@record.id}';"  # Set parent rec id for JS function miqGridSort to build URL
+          page << "miq_parent_class = '#{request[:controller]}';" # Set parent class for URL also
+        end
+        page.replace_html("main_div", :partial => partial, :locals => partial_locals)
+
+      elsif record_showing
+        page.replace_html("main_div", :partial => "container/container_show", :locals => {:controller => "container"})
+      else
+        page.replace_html("main_div",   :partial => "layouts/x_gtl")
+        page.replace_html("paging_div", :partial => "layouts/x_pagingcontrols")
+      end
+
+      page.replace("adv_searchbox_div",
+                   :partial => "layouts/x_adv_searchbox")
+      # Hide/show searchbox depending on if a list is showing
+      page << set_element_visible("adv_searchbox_div", !(@record || @in_a_form))
+
+#     # Decide whether to show paging controls
+      if ["container_edit"].include?(action)
```

Comment:
You can change this to `if "container_edit" == action`.  We don't need the array.